src/gui/optionsdialog.ui

```diff
@@ -2875,8 +2880,8 @@ Disable encryption: Only connect to peers without protocol encryption</string>
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>501</width>
```

Comment:
These changes are unrelated. However, I'm running into the same issue when using the Qt Editor where it changes all of these values whenever I make any other changes as well, so I'm not sure how to prevent it from changing these.

Reply:
Indeed, these changes are introduced by Qt Editor. I could try to manually undo them in the .ui file and check whether it has some effect on the view.

I´ll try that later the day.

Reply:
I was trying to change these values in Qt Creator  but it is not possible. After editing Qt Creator is setting the previous values. All of them seem to be values related to each one of the main widgets from the options tabs.

If I manually edit the .ui file, nothing seems to change, but I guess it is better to trust Qt Creator.

Reply:
>If I manually edit the .ui file, nothing seems to change, but I guess it is better to trust Qt Creator.

Other version/instance of Qt Creator can change it differently so you need to manually revert all unrelated changes after you done your job.

Reply:
> > If I manually edit the .ui file, nothing seems to change, but I guess it is better to trust Qt Creator.
> 
> Other version/instance of Qt Creator can change it differently so you need to manually revert all unrelated changes after you done your job.

Slightly off-topic, but what version do you use that doesn't introduce these new values?

Reply:
>Slightly off-topic, but what version do you use that doesn't introduce these new values?

I didn't say that it doesn't "introduce these new values". I meant that different versions/instances can "introduce different values" so we'll always have them changed each time someone edit `.ui` files.

Reply:
> > Slightly off-topic, but what version do you use that doesn't introduce these new values?
> 
> I didn't say that it doesn't "introduce these new values". I meant that different versions/instances can "introduce different values" so we'll always have them changed each time someone edit `.ui` files.

My bad, I misinterpreted that. Do you use a version that aligns with the current values?

Reply:
> Other version/instance of Qt Creator can change it differently so you need to manually revert all unrelated changes after you done your job.

Should I open a new Pull Request for this?

Reply:
I think I already did that